src/conversion_helper.js

```diff
@@ -73,6 +73,21 @@ function pressureMap(pa) {
   }
 }
 
+function velocityMap(ms) {
+  if (ms <= 56) {
```

Comment:
Oops, small mistake here! The previous behaviour was "If a number is under 200 mph, only convert to km/h" but 56 m/s is 125 mph (201 km/h, I can see how this bug happened XD). This behaviour is here because people generally have a good grasp of mph -> kmh under 200 mph, but over 200 mph, it might gets harder to visualize so the per-second speed is included as well

Perhaps we an also update the numbers in our tests to test the very boundary cases, instead of random numbers so this type of thing is caught in the future?

Reply:
I would dare to say that people who can deal with metric units do have a grasp of velocities <= 200 km/h, maybe the goal should be to have the output numbers in ranges that people can comprehend, instead of the inputs, possibly.

I can change it anyways.

Reply:
Ah, I was just looking at it purely from a refactoring point of view, I didn't even think of "what it should be." 

But you're right, people generally know that 60 mph ~ 100 kmh and 20 mph is "pretty slow for a car" ... If you think that it makes sense to remove conversions for velocities <= 200mph, you can create a new github issue and we can discuss further :D It might be a good "first timers only" issue (I'm working on making the repo more newbie-friendly)

Reply:
I agree. Personally I can deal with km/h and m/s no matter what, but if we wanted to change this, this would be the scope of another issue / pull request.